### PR TITLE
Allow configure tracker for all the teams

### PR DESCRIPTION
**Pull request by jesusfcr**
- Allow to setup `*` for onboarded_teams to activate tracker for all the teams.
- Only for admins or observers

### CODE REVIEW

**Review by seilagamo — CHANGES_REQUESTED** [review]

File: pkg/api/service/tickets_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/adevinta/vulcan-api/pkg/tickets"
+)
+
+func TestVulcanitoService_OnboardeTeam(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (jesusfcr) [reply pkg/api/service/tickets_test.go]: thanks, fixed

File: pkg/api/service/tickets_test.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2021 Adevinta
```
[the remark this answers is not in the document]
Comment (jesusfcr) [reply pkg/api/service/tickets_test.go]: thanks, fixed ;)

**Review by seilagamo — DISMISSED** [review]
LGTM

**Review by seilagamo — APPROVED** [review]
LGTM

**Pull request merged**